src_assets/common/assets/web/config.html

```diff
@@ -448,11 +448,11 @@ <h1 class="my-4">Configuration</h1>
           type="text"
           class="form-control"
           id="audio_sink"
-          placeholder="{0.0.0.00000000}.{FD47D9CC-4218-4135-9CE2-0C195C87405B}"
+          placeholder="Speakers (High Definition Audio Device)"
           v-model="config.audio_sink"
         />
         <div class="form-text">
-          The name of the audio sink used for Audio Loopback<br />
+          The name of the audio sink used for audio capture. If not set, the default audio device will be used.<br />
           You can find the name of the audio sink using the following command:<br />
           <pre>tools\audio-info.exe</pre>
```

Comment:
Could we make the command example show up only when running on Windows? Or make this tool available on Unix platforms as well?

Reply:
Yep, it already does. It's in a `v-if="platform === 'windows'"` div.